tesseract_utils.py

```diff
@@ -1,19 +1,21 @@
-import numpy
-
 from classification import classes_from_numpy, classes_to_numpy
 from segmentation import segments_from_numpy, segments_to_numpy
+import sys
```

Comment:
Please don't override builtins, and don't test the negative case (`==2` vs `!=3`)
AFAICT, `io` works similarly in py2/3?
`import io`
`io.open(path, ...)`